src/main/index.js

```diff
@@ -366,7 +374,11 @@ function runApp() {
       app.commandLine.appendSwitch('enable-smooth-scrolling')
     }
 
-    if (useProxy) {
+    if (useProxy && proxyLogin.length > 0 && proxyPassword.length > 0) {
+      session.defaultSession.setProxy({
+        proxyBypassRules: `${proxyProtocol}://${proxyLogin}:${proxyPassword}@${proxyHostname}:${proxyPort}`
+      })
+    } else {
```

Comment:
Make sure proxy is enabled here or nest the if else in a `if (useProxy)`
```suggestion
    } else if (useProxy) {
```